intl/msg_hash_us.h

```diff
@@ -8299,18 +8527,34 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_VIDEO_SHADER_NUM_PASSES,
    "Increase or decrease the amount of shader pipeline passes. Separate shaders can be bound to each pipeline pass and configure its scale and filtering."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_VIDEO_SHADER_NUM_PASSES,
+   "RetroArch allows you to mix and match various shaders with arbitrary shader passes, with custom hardware filters and scale factors.\nThis option specifies the number of shader passes to use. If you set this to 0, and use Apply Shader Changes, you use a 'blank' shader.\nThe Default Filter option will affect the stretching filter."
```

Comment:
Possible bad inheritance here: at least in Crowdin there is no such option as "Default Filter", only "Default". "Filter" uppercasing should be corrected after confirming it does not exist indeed.

Reply:
Noted, will fix in phase 2 PR.

Reply:
I can't even find the default setting for this option in the actual menu. I'll ask in shader channel, maybe this is changed since it was written.